scripts/ci_tests.sh

```diff
@@ -6,8 +6,15 @@
 #
 # This script could be used for local testing as long as COVERALLS is not set.
 
+set -e
```

Comment:
Instead of this, do we want to run both and then have it report which things failed?

Reply:
`set -ex` probably will be more obvious on which failed

Reply:
done